examplescripts/ubuntu_server.py

```diff
@@ -0,0 +1,181 @@
+from jumpscale.loader import j
+from time import sleep
+import uuid
+import os
+
+bad_nodes = [
+    "J1Xb2piba1vZM2pSmWP24CPgREnoTce7EuUJysttCyz6",
+    "A34YUGenHKyhjDMAUKZe4cVDtJM2wQ4n4XRkfGUUEYdy",
+    "7Qa7fxRPtMc5R72cnNms4XWD9PJkeQJ6iD8iTU9T8p2o",
+    "HYwvrxCy5z1QPALC5gTAiAMYWLH7orQ8xWEDbEyLiJV7",
+    "Dko5HMMaJ9D78rAfgobA3sE29FiVrKst64v2WM4Q52gx",
+    "9AcLUTkRRWGGQBn8T83Qkmuv1mwhjvmJmcvhS6r8Y8Kf",  # times out
+    "HEv8cqTakrSzoxeTyauBiRNomsMFB5N2iqCfHFDY63ct",  # times out
+    "J7twE8kmwkeph5WeKGcDoo1mUFgyYiWANFHtvE79L6Fp",  # times out -> about five minutes y3ne
+    "FVamMByotQxr5azHA9xLoo99qP7FNi1ukysMxpm6apNG",
+    "CBDY1Fu4CuxGpdU3zLL9QT5DGaRkxjpuJmzV6V5CBWg4",  # Bad access node
+]
+DOMAIN = "mydomain.com"
+SUBDOMAIN1 = f"testa.{DOMAIN}"
+SUBDOMAIN2 = f"testb.{DOMAIN}"
+NETWORK_NAME = j.data.random_names.random_name()
+SOLUTION_IP_ADDRESS = "172.18.4.3"
+TRC1_IP_ADDRESS = "172.18.4.4"
+TRC2_IP_ADDRESS = "172.18.4.5"
+SOLUTION_PORT = 3001
+SOLUTION_TLS_PORT = 443  # not serving https actually, but it must be provided
+SECRET = f"{j.core.identity.me.tid}:{uuid.uuid4().hex}"
+
+zos = j.sals.zos
+
+
+def create_pool(currency="TFT", wallet_name=None):
+    wallet_name = wallet_name or os.environ.get("WALLET_NAME")
+    if wallet_name is None:
+        print("Pass WALLET_NAME with the name of the wallet in j.clients.stellar to reserve a pool")
+        exit(0)
+    payment_detail = zos.pools.create(cu=100, su=100, farm="freefarm", currencies=[currency])
+    print(payment_detail)
+
+    wallet = j.clients.stellar.get(wallet_name)
+    txs = zos.billing.payout_farmers(wallet, payment_detail)
+    pool = zos.pools.get(payment_detail.reservation_id)
+    while pool.cus == 0:
+        pool = zos.pools.get(payment_detail.reservation_id)
+        sleep(1)
+
+    print("compute units available:", pool.cus)
```

Comment:
for consistency, could we always use logger?

Reply:
Those prints were only for debugging so I removed them. I also changed the remaining print to use logger, except for the network configuration part to make copy pasting easier.